test/CMakeLists.txt

```diff
@@ -1,13 +1,12 @@
 # Add gtest based cpp test target and link libraries
 find_package(ament_cmake_gtest)
 ament_add_gtest(${PROJECT_NAME}-test 
-    main.cpp
-    test_grid_map_geo.cpp
+  main.cpp
+  test_grid_map_geo.cpp
 )
 
-
 target_link_libraries(${PROJECT_NAME}-test
-    ${PROJECT_NAME}
-        # ${YAML_CPP_LIBRARIES}
-        # ${GDAL_LIBRARY}
+  ${PROJECT_NAME}
+  # ${YAML_CPP_LIBRARIES}
```

Comment:
Let's just remove the commented code.

Reply:
Update in https://github.com/ethz-asl/grid_map_geo/pull/26/commits/a36c8ea6b365f34a0dfdbf2ded4180fb5540ebbe